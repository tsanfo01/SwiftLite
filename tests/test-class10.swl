func foo(x : Int, y : Bool, z : String) -> Int {
    printi(x);
    printb(y);
    print(z);
    return 0;
}

class c {
    init() {
        foo(3, true, "hello");
    }
    func foo(x : Int) -> Int{
        return print("world");
    }
}

var x : c = c();
foo(2, false, "hello");
x.foo(1);