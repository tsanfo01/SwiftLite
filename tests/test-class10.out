3
1
hello
2
0
hello
world
